xml/admin_saltcluster.xml

```diff
@@ -13,707 +13,196 @@
-     <para>
-      Create the new configuration on the &smaster; node:
-     </para>
-<screen>
-&prompt.smaster;salt '<replaceable>SALT_MASTER_NODE</replaceable>' state.apply ceph.configuration.create
-</screen>
-    </step>
-    <step>
-     <para>
-      Apply the new configuration to the targeted OSD minions:
-     </para>
-<screen>
-&prompt.smaster;salt '<replaceable>OSD_MINIONS</replaceable>' state.apply ceph.configuration
-</screen>
-    </step>
-    <step>
-     <para>
-      After the new OSDs are added, adjust their weights as required with the
-      <command>ceph osd crush reweight</command> command.
-     </para>
-    </step>
-   </procedure>
-  </tip>
-
   <procedure>
    <step>
     <para>
-     Install &cephos; on the new node and configure its network setting so that
-     it resolves the &smaster; host name correctly. Verify that it has a proper
-     connection to both public and cluster networks, and that time
-     synchronization is correctly configured. Then install the
-     <systemitem>salt-minion</systemitem> package:
-    </para>
-<screen>&prompt.sminion;zypper in salt-minion</screen>
-    <para>
-     If the &smaster;'s host name is different from <literal>salt</literal>,
-     edit <filename>/etc/salt/minion</filename> and add the following:
-    </para>
-<screen>master: <replaceable>DNS_name_of_your_salt_master</replaceable></screen>
-    <para>
-     If you performed any changes to the configuration files mentioned above,
-     restart the <systemitem>salt.minion</systemitem> service:
-    </para>
-<screen>&prompt.sminion;systemctl restart salt-minion.service</screen>
-   </step>
-   <step>
-    <para>
-     On the &smaster;, accept the salt key of the new node:
+     Install &sls; and &productname; on the new node. Refer to
+     <xref linkend="deploy-os"/> for more information.
     </para>
-<screen>&prompt.smaster;salt-key --accept <replaceable>NEW_NODE_KEY</replaceable></screen>
    </step>
    <step>
     <para>
-     Verify that <filename>/srv/pillar/ceph/deepsea_minions.sls</filename>
-     targets the new &sminion; and/or set the proper &deepsea; grain. Refer to
-     DEAD LINK or
-     DEAD LINKe details.
+     Configure the node as a &sminion; of an already existing &smaster;. Refer
+     to <xref linkend="deploy-salt"/> for more information.
     </para>
    </step>
    <step>
     <para>
-     Run the preparation stage. It synchronizes modules and grains so that the
-     new minion can provide all the information &deepsea; expects.
+     Add the new minion to &cephsalt;, for example:
     </para>
-<screen>&prompt.smaster;salt-run state.orch ceph.stage.0</screen>
-    <important>
-     <title>Possible Restart of &deepsea; stage 0</title>
-     <para>
-      If the &smaster; rebooted after its kernel update, you need to restart
-      &deepsea; stage 0.
-     </para>
-    </important>
-   </step>
-   <step>
+<screen>
+&prompt.smaster;ceph-salt config /ceph_cluster/minions add ses-min5.example.com
```

Comment:
Should also set the `cephadm` role: `ceph-salt config /ceph_cluster/roles/cephadm add ses-min5.example.com`, otherwise this minion will not be added to `ceph orch host ls`